### PR TITLE
Fixed other languages link in Hebrew translated README file, and changed a bit the look of the languages link in all translated files except Portugues

### PR DESCRIPTION
**Pull request by itayweb**

### CODE REVIEW

File: docs/README-he.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+- אין צורך במשתמש
+- אין שום פרסומות
+- אין צורך בלחתום על שום דבר כדי לתרום לפרוייקט
+
+## מידע למשתמש
+https://nuclearmusic.rtfd.io/
+
+## חבילות שנתמכות על ידי הקהילה
+
+לפניכם רשימה של חבילות בשביל שימוש שונים, רובם נתמכות על ידי אדם צד שלישי. אנחנו רוצים להודות לתומכים על עבודתם.
```
Comment (binarica) [inline docs/README-he.md:61]: I would phrase this differently, to emphasize the use for different package managers. Something like:
`לפניכם רשימה של החבילות עבור מנהלי החבילות השונים, רובן מתוחזקות על ידי תומכי צד שלישי`

File: docs/README-he.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+<kbd>[<img title="Português" alt="Português" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/br.svg" width="22">](README-ptbr.md)</kbd>
+<kbd>[<img title="Svenska" alt="Svenska" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/se.svg" width="22">](README-se.md)</kbd>
+<kbd>[<img title="English" alt="English" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/us.svg" width="22">](README.md)</kbd>
+<kbd>[<img title="Hebrew" alt="Hebrew" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/il.svg" width="22">](README-he.md)</kbd>
+<kbd>[<img title="Italiano" alt="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">](README-it.md)</kbd>
+
+## מה זה בכלל Nuclear?
+Nuclear זוהי תוכנה חינמית להזרמת תכנים ממקורות חינמיים ברחבי האינטרנט.
+
+מבינכם שמכירים את [mps-youtube](https://github.com/mps-youtube/mps-youtube), זו תוכנה דומה אך עם תוספת של ממשק משתמש גרפי. אנחנו מתמקדים בעיקר על השמע. דמיינו על ספוטיפיי אבל אתם לא צריכים לשלם עבורו ועם ספרייה גדולה יותר.
```
Comment (binarica) [inline docs/README-he.md:32]: A couple of suggestions about מילות היחס:
`אנחנו מתמקדים בעיקר על השמע` -> `אנחנו מתמקדים בעיקר באודיו`
`דמיינו על ספוטיפיי` -> `דמיינו את ספוטיפיי`

And one about making the wording a bit more colloquial:
`אבל אתם לא צריכים לשלם עבורו` -> `אבל ללא תשלום`

**Review by binarica — CHANGES_REQUESTED** [review]

**Comment by nukeop** [comment]
Thanks for reviewing the change @binarica ! Please let me know when it's ready to merge.

**Comment by itayweb** [comment]
> Thanks for reviewing the change @binarica ! Please let me know when it's ready to merge.

Fixed the words, the commit is ready to merge

**Comment by HaiDang666** [comment]
I think you should not commit package-lock.json and also rebase master branch to resolve the conflicts

**Comment by itayweb** [comment]
> rebase master branch

When I added the changes it doesn't say that the package-lock.json file was changed, and what is rebasing master branch and how to do that?

**Comment by nukeop** [comment]
Thanks, those little flags look very nice. I can let `package-lock.json` stay because it will have no net changes, but in the future please do not commit it. There are some merge conflicts but i think I will be able to resolve them manually. Thanks for contributing.

**Comment by itayweb** [comment]
> Thanks, those little flags look very nice. I can let `package-lock.json` stay because it will have no net changes, but in the future please do not commit it. There are some merge conflicts but i think I will be able to resolve them manually. Thanks for contributing.

Alright, sorry for the troubles, I will do it next time

**Pull request merged**